<?xml version="1.0"?>
<launch>
    <arg name="openpose_models_dir" value="/home/lucy/.models/op_models/" />
    <arg name="pointcloud_topic" default="/mdr_perception/rectified_points" />
    <arg name="ssd_detector_class_file" default="$(find ssd_keras_ros)/models/coco_classes.yml" />
    <arg name="ssd_detector_kwargs_file" default="$(find ssd_keras_ros)/models/ssd_keras_object_detector_kwargs.yml" />

    <node pkg="mdr_recognize_gesture_action" type="recognize_gesture_action" name="mdr_recognize_gesture_server" output="screen">
        <param name="gesture_type" value="pointing" />
        <param name="openpose_models_dir" type="str" value="$(arg openpose_models_dir)" />
        <param name="pointcloud_topic" type="str" value="$(arg pointcloud_topic)" />
        <param name="ssd_detector_class_file" type="str" value="$(arg ssd_detector_class_file)" />
        <param name="ssd_detector_kwargs_file" type="str" value="$(arg ssd_detector_kwargs_file)" />
    </node>
</launch>
